demo.sh

```diff
@@ -21,13 +21,16 @@ if [[ -n "$APOLLO_PORTAL_DB_PASSWORD" ]]; then
   echo APOLLO_PORTAL_DB_PASSWORD = "${APOLLO_PORTAL_DB_PASSWORD//?/*}"
 fi
 
+# database platform
+spring_profiles_group_github=${SPRING_PFOILES_GROUP_GITHUB:-mysql}
+
 # apollo config db info
-apollo_config_db_url="jdbc:mysql://localhost:3306/ApolloConfigDB?characterEncoding=utf8&serverTimezone=Asia/Shanghai"
+apollo_config_db_url=${APOLLO_CONFIG_DB_URL:-"jdbc:mysql://localhost:3306/ApolloConfigDB?characterEncoding=utf8&serverTimezone=Asia/Shanghai"}
```

Comment:
How about we add an echo statement to print the db url just like what we did to `APOLLO_CONFIG_DB_USERNAME`?

Reply:
@nobodyiam Fixed